pkg/operator/telemetry/telemetry.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2019 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package telemetry
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/cortexlabs/cortex/pkg/consts"
+	cr "github.com/cortexlabs/cortex/pkg/lib/configreader"
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/operator/aws"
+	cc "github.com/cortexlabs/cortex/pkg/operator/cortexconfig"
+)
+
+const defaultTelemetryURL = "https://telemetry.cortexlabs.com"
+const eventPath = "/events"
+const errorPath = "/errors"
+
+var telemetryURL string
```

Comment:
I think this and `httpClient` should be fields in a `type Client struct`package variable you export